bitcoin/src/blockdata/script/tests.rs

```diff
@@ -311,23 +312,23 @@ fn scriptint_round_trip() {
         -((1 << 31) - 1),
     ];
     for &i in test_vectors.iter() {
-        assert_eq!(Ok(i), read_scriptint(&build_scriptint(i)));
-        assert_eq!(Ok(-i), read_scriptint(&build_scriptint(-i)));
+        assert_eq!(Ok(i), PushBytes::read_scriptint(&build_scriptint(i)));
+        assert_eq!(Ok(-i), PushBytes::read_scriptint(&build_scriptint(-i)));
```

Comment:
> This must take &self there's no point of this change otherwise.

If `read_scriptint` function would take `&self` then  how we can make this thing run as `read_scriptint` would then need   the struct instance but we don't have any public method to create `PushBytes` instance other than having `PushBytes::empty()` method .